config/config.go

```diff
@@ -101,11 +102,19 @@ func Environment() string {
 	return env
 }
 
-// IsDevelopment returns true if the current environment is set to development
-func IsDevelopment() bool {
+// IsDevelopmentEnv returns true if the current environment is set to development
+func IsDevelopmentEnv() bool {
 	return strings.Contains(Environment(), _devEnv)
 }
 
+// IsTestEnv returns true if the current environment is set to test
+//
+// Useful, for example, to enable alternate functionality of some components
+// during testing that might normally require network connectivity or a database
+func IsTestEnv() bool {
```

Comment:
Based on the earlier discussion, I'll make some changes to this diff and remove this functionality.